SPECS/cal10n/cal10n.spec

```diff
@@ -25,7 +25,7 @@ Summary:        Compiler assisted localization library (CAL10N)
 License:        MIT
```

Comment:
The newest version is 0.8.1.  
https://github.com/qos-ch/cal10n/releases/tag/v_0.8.1

Reply:
Upgrading will require to switch to a different build method, will do in another PR.